### ISSUE TITLE
Dependency Dashboard

### ISSUE THREAD

**Issue by lne-bamboo** [milieuinfo/webcomponenten-vl-ui-bom#13]
This issue lists Renovate updates and detected dependencies. Read the [Dependency Dashboard](https://docs.renovatebot.com/key-concepts/dashboard/) docs to learn more.

---

### ⚠ Dependency Lookup Warnings ⚠

-   Renovate failed to look up the following dependencies: `vl-ui-accessibility`, `vl-ui-accordion`, `vl-ui-alert`, `vl-ui-body`, `vl-ui-breadcrumb`, `vl-ui-button`, `vl-ui-checkbox`, `vl-ui-code-preview`, `vl-ui-contact-card`, `vl-ui-content-header`, `vl-ui-cookie-consent`, `vl-ui-cookie-statement`, `vl-ui-core`, `vl-ui-data-table`, `vl-ui-datepicker`, `vl-ui-demo`, `vl-ui-description-data`, `vl-ui-document`, `vl-ui-doormat`, `vl-ui-footer`, `vl-ui-form`, `vl-ui-form-grid`, `vl-ui-form-message`, `vl-ui-functional-header`, `vl-ui-grid`, `vl-ui-header`, `vl-ui-http-error-message`, `vl-ui-icon`, `vl-ui-image`, `vl-ui-info-tile`, `vl-ui-infoblock`, `vl-ui-infotext`, `vl-ui-input-addon`, `vl-ui-input-field`, `vl-ui-input-group`, `vl-ui-introduction`, `vl-ui-link`, `vl-ui-link-list`, `vl-ui-loader`, `vl-ui-map`, `vl-ui-modal`, `vl-ui-multiselect`, `vl-ui-pager`, `vl-ui-pattern`, `vl-ui-pill`, `vl-ui-privacy`, `vl-ui-progress-bar`, `vl-ui-properties`, `vl-ui-proza-message`, `vl-ui-radio`, `vl-ui-rich-data`, `vl-ui-rich-data-table`, `vl-ui-search`, `vl-ui-search-filter`, `vl-ui-search-results`, `vl-ui-side-navigation`, `vl-ui-side-sheet`, `vl-ui-steps`, `vl-ui-tabs`, `vl-ui-template`, `vl-ui-textarea`, `vl-ui-titles`, `vl-ui-toaster`, `vl-ui-tooltip`, `vl-ui-typography`, `vl-ui-upload`, `vl-ui-util`, `vl-ui-video-player`, `vl-ui-wizard`, `copyfiles`, `mocha-junit-reporter`.

Files affected: `package.json`

---

## Open

These updates have all been created already. Click a checkbox below to force a retry/rebase of any.

 - [ ] [Roll back dependency mocha-multi-reporters to 1.0.0-art-external1](../pull/164)
 - [ ] [Roll back dependency vl-ui-action-group to 1.0.0-art-external1](../pull/165)
 - [ ] [Roll back dependency vl-ui-form-validation to 1.0.0-art-external1](../pull/166)
 - [ ] [Roll back dependency vl-ui-select to 1.0.0-art-external1](../pull/170)
 - [ ] **Click on this checkbox to rebase all open PRs at once**

## Detected dependencies

<details><summary>npm</summary>
<blockquote>

<details><summary>package.json</summary>

 - `vl-ui-accessibility 1.0.0`
 - `vl-ui-accordion 3.1.4`
 - `vl-ui-action-group 3.1.4`
 - `vl-ui-alert 4.1.2`
 - `vl-ui-body 1.0.8`
 - `vl-ui-breadcrumb 1.0.4`
 - `vl-ui-button 5.0.11`
 - `vl-ui-checkbox 3.3.0`
 - `vl-ui-code-preview 1.0.5`
 - `vl-ui-contact-card 1.0.2`
 - `vl-ui-content-header 3.0.7`
 - `vl-ui-cookie-consent 4.1.3`
 - `vl-ui-cookie-statement 1.0.2`
 - `vl-ui-core 7.2.4`
 - `vl-ui-data-table 4.0.5`
 - `vl-ui-datepicker 3.2.8`
 - `vl-ui-demo 1.3.3`
 - `vl-ui-description-data 3.0.7`
 - `vl-ui-document 1.0.3`
 - `vl-ui-doormat 1.0.3`
 - `vl-ui-footer 3.4.2`
 - `vl-ui-form 1.1.3`
 - `vl-ui-form-grid 3.0.10`
 - `vl-ui-form-message 5.1.1`
 - `vl-ui-form-validation 4.0.2`
 - `vl-ui-functional-header 1.2.12`
 - `vl-ui-grid 3.3.1`
 - `vl-ui-header 3.4.4`
 - `vl-ui-http-error-message 3.2.3`
 - `vl-ui-icon 5.2.3`
 - `vl-ui-image 3.0.7`
 - `vl-ui-info-tile 1.1.4`
 - `vl-ui-infoblock 3.0.11`
 - `vl-ui-infotext 3.0.7`
 - `vl-ui-input-addon 3.1.10`
 - `vl-ui-input-field 3.2.12`
 - `vl-ui-input-group 4.0.8`
 - `vl-ui-introduction 1.0.6`
 - `vl-ui-link 4.1.1`
 - `vl-ui-link-list 3.0.9`
 - `vl-ui-loader 1.0.6`
 - `vl-ui-map 7.1.7`
 - `vl-ui-modal 4.0.5`
 - `vl-ui-multiselect 4.2.1`
 - `vl-ui-pager 3.0.12`
 - `vl-ui-pattern 1.2.1`
 - `vl-ui-pill 4.1.0`
 - `vl-ui-privacy 1.0.2`
 - `vl-ui-progress-bar 1.1.4`
 - `vl-ui-properties 4.1.0`
 - `vl-ui-proza-message 1.3.0`
 - `vl-ui-radio 1.2.2`
 - `vl-ui-rich-data 2.1.2`
 - `vl-ui-rich-data-table 2.0.1`
 - `vl-ui-search 1.1.11`
 - `vl-ui-search-filter 3.3.4`
 - `vl-ui-search-results 2.0.0`
 - `vl-ui-select 4.3.3`
 - `vl-ui-side-navigation 1.0.1`
 - `vl-ui-side-sheet 3.2.0`
 - `vl-ui-steps 1.2.0`
 - `vl-ui-tabs 1.1.0`
 - `vl-ui-template 3.2.1`
 - `vl-ui-textarea 3.3.4`
 - `vl-ui-titles 3.1.2`
 - `vl-ui-toaster 3.0.9`
 - `vl-ui-tooltip 3.0.10`
 - `vl-ui-typography 3.1.6`
 - `vl-ui-upload 3.7.3`
 - `vl-ui-util 5.6.0`
 - `vl-ui-video-player 1.0.2`
 - `vl-ui-wizard 1.3.2`
 - `copyfiles 2.4.1`
 - `mocha-junit-reporter 2.2.0`
 - `mocha-multi-reporters 1.5.1`

</details>

</blockquote>
</details>
